### PR TITLE
Adds factory and new-style config for Logging exporter.

### PR DESCRIPTION
**Pull request by dinooliva**
Fix for issue #37

### CODE REVIEW

File: exporter/loggingexporter/config.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+package loggingexporter
+
+import (
+	"github.com/open-telemetry/opentelemetry-service/models"
+)
+
+// ConfigV2 defines configuration for logging exporter.
+type ConfigV2 struct {
+	models.ExporterSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct.
```
Comment (songy23) [inline exporter/loggingexporter/config.go]: Nit: consider adding a TODO to allow configure logger levels.

**Comment by tigrannajaryan** [comment]
>Adds factory and new-style config for Logging exporter. 

Nit 1: Adds -> Add (Use the imperative mood in the subject line).
Nit 2: Do not end the subject line with a period.

Source: see reference to https://chris.beams.io/posts/git-commit/ in https://github.com/open-telemetry/community/blob/master/CONTRIBUTING.md

**Comment by bogdandrutu** [comment]
@tigrannajaryan do we have static analysis tools to detect these issues?

**Comment by tigrannajaryan** [comment]
>do we have static analysis tools to detect these issues?

@bogdandrutu I don't think we do. Would be good to automate this. Sort of a linter for commit messages.

**Comment by tigrannajaryan** [comment]
Looks like such a tool exists: https://github.com/jorisroovers/gitlint

**Comment by dinooliva** [comment]
Acknowledged - interesting read.

File: exporter/loggingexporter/config.go (hunk 1 above)
Comment (dinooliva) [reply exporter/loggingexporter/config.go]: Done.

**Pull request merged**